src/app/[locale]/page.tsx

```diff
@@ -63,6 +67,29 @@ const Profile = () => {
     }
   }, [callFetchWithRetries, isProfileAvailable, pushWithLocale]);
 
+  useEffect(() => {
+    if (isProfileAvailable) {
+      callFetchWithRetries(WebProfileApi, 'readInfo', [], [500])
+        .then((res) => {
+          const mockWalletStatus = global.window?.localStorage?.getItem("walletStatus") // todo: rimuovere una volta che sarà funzionante l'api di revoke
+          setWalletData(mockWalletStatus ? { status: mockWalletStatus } : res);
+        })
+        .catch(() => pushWithLocale(ROUTES.INTERNAL_ERROR));
+    }
+  }, [isProfileAvailable]);
```

Comment:
React Hook useEffect has missing dependencies: 'callFetchWithRetries' and 'pushWithLocale'.